### PR TITLE
Basic readme file added.

### LINKED ISSUE #45: Create a Readme
https://github.com/Appsilon/readme_rmd_template

### PR DESCRIPTION
**Pull request by maju116**
Closes #45

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- infoBox
+- infoBoxOutput
+- messageItem
+- notificationItem
+- renderDropdownMenu
+- renderInfoBox
+- tabBox
+- taskItem
+
+[Unreleased]: https://github.com/Appsilon/your_repo_url/compare/0.1.0...HEAD
```
Comment (filipstachura) [inline CHANGELOG.md]: Fix url

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+[0.1.0]: https://github.com/Appsilon/your_repo_url/compare/your_repo_init_comit_hash...0.1.0
```
Comment (filipstachura) [inline CHANGELOG.md]: Fix url

File: README.Rmd [hunk 3]
```diff
@@ -0,0 +1,112 @@
+---
+output: html_document
+params:
+   website_url: http://appsilondatascience.com
+   logo_url: https://cdn.rawgit.com/Appsilon/website-cdn/gh-pages/logo-white.png
+   repo_name: semantic.dashboard
+   repo_url: http://appsilondatascience.com/repo
```
Comment (filipstachura) [inline README.Rmd]: Fix url

File: README.Rmd [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+   demo_url: http://appsilondatascience.com/demo
```
Comment (filipstachura) [inline README.Rmd]: No demo yet - please remove and add Asana task to add this one.

File: README.Rmd [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+<div class="subheader">Dashboard with Semantic UI Support for Shiny</div>
+
+<div class="section level2">
+Basic functions for creating dashboard with Semantic UI.
+
+<!-- #Basic tutorial article is available on [Appsilon Data Science blog](your_future_art_link). -->
+
+<!-- Live demo link below -->
+<p style="text-align: center; font-size: x-large;">
```
Comment (filipstachura) [inline README.Rmd]: Please comment for now.

File: CHANGELOG.md (hunk 1 above)
Comment (maju116) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 2 above)
Comment (maju116) [reply CHANGELOG.md]: done

File: README.Rmd (hunk 3 above)
Comment (maju116) [reply README.Rmd]: done

File: README.Rmd (hunk 4 above)
Comment (maju116) [reply README.Rmd]: done

File: README.Rmd (hunk 5 above)
Comment (maju116) [reply README.Rmd]: done

**Pull request merged**